docs/docusaurus/docs/components/_data.jsx

```diff
@@ -1,5 +1,5 @@
 export default {
   release_version: 'great_expectations, version 0.17.6',
-  min_python: '3.7',
-  max_python: '3.10'
+  min_python: '3.8',
+  max_python: '3.11'
 }
```

Comment:
what happened to this change?

Reply:
@Kilo59 it appears that this change was implemented in another PR as it is showing in the file:

![image](https://github.com/great-expectations/great_expectations/assets/104205257/bd06aea6-3969-41fc-918a-e92b18f17b59)

Reply:
Thanks